### PR TITLE
make relocatable

### PR DESCRIPTION
**Pull request by jbrea**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+desc = "Entities from HTML data tables"
+authors = ["ScottPJones <scottjones@alum.mit.edu>"]
 version = "1.0.1"
 
 [deps]
+RelocatableFolders = "05181044-ff0b-4ac5-8273-598c1e38db00"
 StrTables = "9700d1a9-a7c8-5760-9816-a99fda30bb8f"
 
+[compat]
+RelocatableFolders = "0.3"
```
Comment (MichaelHatherly) [inline Project.toml]: If using `RelocatableFolders` for this one, I'd suggest going with `1` compat. Looks like this PR was made at pretty much the exact same time as version 1.0.0 was released. No difference in exposed API, it's just considered feature-complete and stable.

```suggestion
RelocatableFolders = "1"
```

**Comment by andyferris** [comment]
Today I have been bitten because HTML_Entities.jl isn't relocatable (as a part of a much larger docker build).

Is there any likelihood of landing this or something similar?

**Comment by JackDunnNZ** [comment]
Bump, would be good to fix this

**Comment by ScottPJones** [comment]
Sorry I missed this for so long!

**Comment by AbrJA** [comment]
@JuliaRegistrator register()

**Comment by ScottPJones** [comment]
@JuliaRegistrator register()

**Pull request merged**